package.json

```diff
@@ -70,7 +70,7 @@
     "enzyme-to-json": "^3.3.4",
     "eslint": "^4.15.0",
     "eslint-config-airbnb": "^16.1.0",
-    "eslint-config-sonar": "0.0.6",
+    "eslint-config-sonar": "0.0.7",
```

Comment:
wondering if we should put a hat on this

Reply:
hat wouldn't make a difference at 0.0.x as every change is a breaking change

Reply:
ah - true.

perhaps that means `eslint-config-sonar` should get it's version updated to `0.1.0`?